src/cow-react/modules/limitOrders/utils/toFirstMeaningfulDecimal.ts

```diff
@@ -0,0 +1,21 @@
+export function toFirstMeaningfulDecimal(value: string | undefined | null, limit = 6): string {
```

Comment:
Why do we need this? 
I believe this is already provided by BigDecimal and so on, and actually this implementations will fail for other locales

Reply:
There is `toSignificant` method on Fractions, so we can use that instead